OutOfSchool/OutOfSchool.WebApi/appsettings.Release.json

```diff
@@ -144,6 +144,9 @@
   },
   "ElasticApm": {
     "Enabled": true
+  },
+  "InstitutionOptions": {
+    "MinistryOfSportId": "b67a4f29-728e-4bb0-bb42-4a9d7e0bd90a"
```

Comment:
as we use same value, no need to repeat everywhere, regular `appsettings.json` is fine